automatilib/cola/views.py

```diff
@@ -28,11 +30,19 @@
     "COLA_COOKIE_NAME",
     "LOGIN_REDIRECT_URL",
     "LOGIN_URL",
+    "LOGIN_FAILURE_TEMPLATE_PATH",
+    "CONTACT_EMAIL",
 )
 for setting_key in SETTINGS:
     if not getattr(settings, setting_key):
         raise ImproperlyConfigured(f"{setting_key} must be set")
 
+try:
```

Comment:
could/should this be a one-time operation that raises a `ImproperlyConfigured` error if it cant find the template?